README.md

```diff
@@ -1,2 +1,107 @@
+* **[`eslint-config-udemy-website`](packages/eslint-config-udemy-website):**
+The ESLint configuration used by 
+[`udemy/website-django/static/`](https://github.com/udemy/website-django/tree/master/static/.eslintrc.js) codebase.
+* **[`eslint-plugin-udemy`](packages/eslint-plugin-udemy):**
+A set of custom ESLint rules written for Udemy, by Udemy. This rules are mainly used by  
+[`eslint-config-udemy-website`](packages/eslint-config-udemy-website).
+
+# Contributing
+
+Install [*Yarn@1.3.2*](https://github.com/yarnpkg/yarn/releases/tag/v1.3.2) globally. 
+
+    $ npm install -g yarn@1.3.2
+    
+<small>**Note:** Since our main `udemy/website-django` codebase
+uses a different Yarn version at the moment (v0.22), you might want to utilize 
+[*nvm*](https://github.com/creationix/nvm) to maintain multiple Yarn versions. e.g. Use 
+`nvm use system && npm install -g yarn@0.22` for `udemy/website-django`, and 
+`nvm install 6.10.1 && nvm use 6.10.1 && npm install -g yarn@1.3.2` for `udemy/eslint-udemy`. 
+If you do so, you need to remember to switch between `6.10.1` vs `system` whenever you are working on
+one project or the other. This is a temporary measure until 
+[@udemy/sre](https://github.com/orgs/udemy/teams/sre) unifies Yarn versions across the board.</small>
+
+Install all dependencies locally.
+
+    $ `yarn bin`/lerna bootstrap
+    
+Run tests to verify everything is working.
+
+    $ npm run test
+
+### Updating an existing package
+
+1. Create a new branch.
+1. Add any new dependencies to any of the packages via `lerna add`.
+1. Run `lerna bootstrap` to make sure everything is installed.
+1. Make your necessary source file changes.
+1. Write your tests if applicable.
+1. Do *NOT* update version numbers in `package.json` files (`lerna publish` does it for you).
+1. Run `npm run test` to make sure all tests pass.
+1. Commit/push your changes.
+1. Create a pull request against master.
+1. Once your pull request is approved, merge it to master.
+1. Pull latest master locally.
+1. Run `lerna publish` in order to publish your changes to npm.
+1. Go to the repository where you'd use these new ESLint rule changes.
+1. Update the `package.json` dependencies to any `eslint-config|plugin-udemy-*` package as necessary.
+
+You can always reach out to [@udemy/team-f](https://github.com/orgs/udemy/teams/team-f) at 
+[#dev-team-f Slack channel](https://udemy.slack.com/messages/dev-team-f).
+ 
+### Adding a new package
+
+1. Get in touch with [@udemy/team-f](https://github.com/orgs/udemy/teams/team-f) at 
+[#dev-team-f Slack channel](https://udemy.slack.com/messages/dev-team-f) to assess the need for a
+new npm package to avoid any duplicate work.
+1. Once there is consensus, create a new folder under [packages](packages/) folder with the
+desired npm package name.
+1. Use `index.js` as package's main entry point, `tests.js` as its entry point for tests.
+1. Make sure `package.json` has necessary information, following existing examples.
```

Comment:
has the

Reply:
per the existing examples.

Reply:
Fricking articles :D

Reply:
I completely agree! Italian is worse since they almost always use articles, and there are female vs. male, singular vs. plural variations ;)